test/integration/controllermanager/shoot/maintenance/maintenance_suite_test.go

```diff
@@ -111,10 +113,12 @@ var _ = BeforeSuite(func() {
 	mgrClient = mgr.GetClient()
 
 	By("registering controller")
+	fakeClock = &testclock.FakeClock{}
```

Comment:
Maybe initialize it with `time.Now()`?
To avoid issues like https://github.com/gardener/gardener/pull/6619/files.